src/zope/server/maxsockets.py

```diff
@@ -11,80 +11,24 @@
 # FOR A PARTICULAR PURPOSE.
 #
 ##############################################################################
-"""Find max number of sockets allowed.
 """
-# Medusa max_sockets module.
+Find max number of sockets allowed.
+
+.. caution:: These functions are all hard coded to return a constant value
+   of 100. Attempting to dynamically compute them by looping and creating
+   sockets until failure was very time consuming.
```

Comment:
The original comment gave me the impression it was very time consuming *on MacOS X*.  Maybe let's preserve that bit of information?

Reply:
gevent had or had something similar and I observed it to be slow (and buggy) everywhere, or at least both locally on OS X and on Travis containers.